src/main/java/co/omise/models/Balance.java

```diff
@@ -4,49 +4,69 @@
 import co.omise.requests.RequestBuilder;
 import co.omise.requests.ResponseType;
 import okhttp3.HttpUrl;
+import okhttp3.RequestBody;
+
+import java.io.IOException;
 
 /**
- * Represents Omise Balance object and contains its {@link RequestBuilder}
+ * Balance object
  *
  * @see <a href="https://www.omise.co/balance-api">Balance API</a>
  */
 public class Balance extends Model {
-    private long transferable;
-    private long total;
     private String currency;
+    private String location;
+    private long reserve;
+    private long total;
+    private long transferable;
 
-    public Balance() {
+    public String getCurrency() {
+        return this.currency;
     }
 
-    public long getTransferable() {
-        return transferable;
+    public void setCurrency(String currency) {
+        this.currency = currency;
     }
 
-    public void setTransferable(long transferable) {
-        this.transferable = transferable;
+    public String getLocation() {
+        return this.location;
+    }
+
+    public void setLocation(String location) {
+        this.location = location;
+    }
+
```

Comment:
I hate these get/set methods in Java more than anything. Why not just make everything public?
Sorry, no need to change anything, I'm just ranting :D

Reply:
@nikolack Yeah, it doesn't convenient. But I think we should follow Java code conventional. 🙂